docs/controls/data-management/grid/pdf-export.md

```diff
@@ -0,0 +1,253 @@
+
+###### Example - enable PDF export
+
+```html
+    <!-- Load Pako Deflate library to enable PDF compression -->
+    <script src="http://kendo.cdn.telerik.com/{{ site.cdnVersion }}/js/pako_deflate.min.js"></script>
+
+    <div id="grid"></div>
+    <script>
+        $("#grid").kendoGrid({
+            toolbar: ["pdf"],
+            pdf: {
+                fileName: "Kendo UI Grid Export.pdf"
+            },
+            dataSource: {
+                type: "odata",
+                transport: {
+                    read: "http://demos.telerik.com/kendo-ui/service/Northwind.svc/Products"
+                },
+                pageSize: 7
+            },
+            sortable: true,
+            pageable: true,
+            columns: [
+                { width: 300, field: "ProductName", title: "Product Name" },
+                { field: "UnitsOnOrder", title: "Units On Order" },
+                { field: "UnitsInStock", title: "Units In Stock" }
+            ]
+        });
+    </script>
+```
+
+To initiate PDF export via code, call the [`saveAsPdf`](/api/javascript/ui/grid.html#methods-saveAsPdf) method.
+
+> **Important**
+> * By default, Kendo UI Grid exports the current page of the data with sorting, filtering, grouping, and aggregates applied.
+> * The Grid uses the current column order, visibility, and dimensions to generate the PDF file.
+
+## Features
+
+### Export all Pages
+
+By default, the Kendo UI Grid exports only the current page of data. To export all pages set the [`allPages`](/api/javascript/ui/grid#configuration-pdf.allPages) option to `true`.
+
+> **Important**
+> When the `allPages` option is set to `true` and `serverPaging` is enabled, the Grid will make a `"read"` request for all data. If the data items are too many the browser may become unresponsive. Consider implementing server-side export for such cases.
+
+###### Example - export all pages
+
+```html
+    <div id="grid"></div>
+    <script>
+        $("#grid").kendoGrid({
+            toolbar: ["pdf"],
+            pdf: {
+                allPages: true
+            },
+            dataSource: {
+                type: "odata",
+                transport: {
+                    read: "http://demos.telerik.com/kendo-ui/service/Northwind.svc/Products"
+                },
+                pageSize: 7
+            },
+            pageable: true,
+            columns: [
+                { width: 300, field: "ProductName", title: "Product Name" },
+                { field: "UnitsOnOrder", title: "Units On Order" },
+                { field: "UnitsInStock", title: "Units In Stock" }
+            ]
+        });
+    </script>
+```
+
+### Server Proxy
+
+Internet Explorer 9, or older, and Safari do not support the option for saving a file and require the implementation of a [server proxy](/framework/save-files/introduction#browser-support). Set the [`proxyURL`](/api/javascript/ui/grid#configuration-pdf.proxyURL) option to specify the server proxy URL.
```

Comment:
*“or older”* might trick one to believe that PDF export is supported in IE < 9, which is not the case. ;-)  I'd remove that.